src/librbd/cache/pwl/Types.h

```diff
@@ -150,6 +150,16 @@ enum {
   l_librbd_pwl_last,
 };
 
+enum {
+  write_log_cache_entry_valid = 1U << 0,      /* if 0, this entry is free */
```

Comment:
Please capitalize these:
```suggestion
  WRITE_LOG_CACHE_ENTRY_VALID = 1U << 0,      /* if 0, this entry is free */
```